### DISCUSSION TITLE
[Question]: Windows 11 Pro, Virtualbox 7.0, Frigate, Google Coral TPU.   Is this possible ?

### DISCUSSION THREAD

**Discussion by jorge-rutllant** [blakeblackshear/frigate#12891] [Ask A Question]
### What is your question:

Dear all, I am trying to configure the following:

   - Windows 11 Pro running an I7 (16 threads) with 64 GB Ram as the main machine
   - Virtual Box 7.0 --> Ubuntu 22.04 with 4 cores and 8 GB Ram
   - Frigate inside the Ubuntu VM using Docker Compose
   - Coral TPU passed to the Ubuntu VM

I know this is not a recommended approach but I am trying not to have yet another box just for this.

So far I have managed to install and make Frigate work fine without the Coral TPU.  By this I mean I can see and work with frigate UI and the cameras fine.

My problem is to make the Ubuntu VM access the Coral TPU.

In windows device manager I see “Coral USB Accelerator” as a Serial Bus Device.

In Virtual Box settings for the Ubuntu VM I see “Google Inc.  [0100]” which I select and pass to the VM.

But inside the Ubuntu VM when using “sudo lsusb” I only see:
   Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
   Bus 001 Device 002: ID 80ee:0021 VirtualBox USB Tablet
   Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub

I have followed some guides without success.

Has anybody here being able to make something like this configuration work ?.

If yes, could you please tell me how can I move forward or is this endeavor definitely doomed.

Many Thanks,
Jorge.

**Comment by NickM-27** [comment]
this is not the right way to go about it, using WSL in windows would be a better way
